econplayground/main/models.py

```diff
@@ -474,3 +474,87 @@ class Meta:
 
     created_at = models.DateTimeField(auto_now_add=True)
     updated_at = models.DateTimeField(auto_now=True)
+
+
+class Question(models.Model):
+    adaptive = models.BooleanField(default=False)
+    ap_correct = models.ForeignKey(
+        'self',
+        on_delete=models.SET_NULL,
+        null=True,
+        blank=True,
+        related_name='ap_continue'
+    )
+    ap_incorrect = models.ForeignKey(
+        'self',
+        on_delete=models.SET_NULL,
+        null=True,
+        blank=True,
+        related_name='ap_intervene'
+    )
+    title = models.CharField(max_length=256, default='Untitled', unique=True)
+
+    embedded_media = models.TextField(blank=True, default='')
+    graph = models.ForeignKey(
+        Graph, on_delete=models.CASCADE, blank=True)
+
+    is_assessment = models.BooleanField(default=True)
+    is_key = models.BooleanField(default=True)
+    prompt = models.TextField(blank=True, default='')
+    instructors = models.ManyToManyField(User)
```

Comment:
I don't think we need to tie the question to the instructor at all yet - let's just remove this field for now. Keep in mind - although Tom talks big on his ideas here, in practice he is the only instructor using EconPractice right now.

We can treat the Questions as a global question bank, the instructor that authored the question isn't really important yet.

Reply:
Understood